### DISCUSSION TITLE
Hide Date label in Time axis

### DISCUSSION THREAD

**Discussion by pryvyd9** [epezent/implot#528] [Q&A]
How to hide date from time axis? I assume it's a major tick label but I can't seem to find where to disable it.
![зображення](https://github.com/epezent/implot/assets/26070759/e613d7bd-15e9-4c38-8e2f-d56b25db14f5)

**Comment by Nahor** [comment]
The date is not always a major tick. It depends on your zoom level, e.g.:
![image](https://github.com/epezent/implot/assets/1198364/0d270a3f-dfae-4898-94b3-de516d6d87be)
Major tick every 1h, minor every 30min

Or:
![image](https://github.com/epezent/implot/assets/1198364/a3df14b8-0b3b-4367-9510-e217a277f0ed)
Major every month, minor every week.

And I'm interested in removing the date as well. I started looking at using custom ticks (i.e. computing the intervals myself instead of using `ImPlotScale_Time`). That works OK, except that I don't think it's possible to set major ticks in that case, only the minor ones.
